nixos/modules/hardware/temp-throttle.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
```

Comment:
You need to add your module to the [module-list.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/module-list.nix) for it to work.